packages/cli/src/util/deploy/get-deployments-by-project-id.ts

```diff
@@ -1,26 +1,47 @@
 import { URLSearchParams } from 'url';
-import { Alias } from '../../types';
+import { Deployment } from '../../types';
 import Client from '../client';
 
-type Response = {
-  deployments: DeploymentPartial[];
-};
-export interface DeploymentPartial {
-  id: string;
-  name: string;
-  url: string;
+type LegacyDeployment = {
```

Comment:
**suggestion (non-blocking):** change type name to be more specific

What does "legacy" mean, specifically? It looks like this was an unnamed `version: 1` and the current `Deployment` is `version: 2`, right?

If that's correct, can we call this `DeploymentV1` and the modern version `DeploymentV2` to keep them clear?